app-config-exec/src/index.test.ts

```diff
@@ -0,0 +1,120 @@
+import { defaultEnvExtensions, defaultExtensions, loadUnvalidatedConfig } from '@lcdev/app-config';
+import execParsingExtension from '.';
+
+const defaultOptions = {
+  environmentExtensions: defaultEnvExtensions().concat(execParsingExtension()),
+  parsingExtensions: defaultExtensions().concat(execParsingExtension()),
+};
+
+describe('execParsingExtension', () => {
+  it('reads from command as root level string', async () => {
+    process.env.APP_CONFIG = JSON.stringify({
```

Comment:
I have a `beforeEach` in app-config's tests that reset `process.env` (I think it's in setup-tests). Might be worth adding here as well so that tests are truly isolated.